### PR TITLE
Parser now Handles Booleans

### PR DESCRIPTION
**Pull request by Hopding**
Updates the UI to handle the parser update for https://github.com/maryvilledev/codesplain/issues/17.

screen shot 2017-03-16 at 8 06 24 pm

### CODE REVIEW

File: src/util/codemirror-utils.js [hunk 1]
```diff
@@ -9,13 +9,28 @@ function highlightNode(codeMirror, node, filters, parentColor) {
   let color = parentColor;
 
   // If we aren't ignoring this token...
-  if (ignoredRules.indexOf(node.type) === -1  && rules[node.type]) {
-    color = rules[node.type].color; // Get the color for this token's type
+// <<<<<<< HEAD
```
Comment (solkaz) [inline src/util/codemirror-utils.js]: Please remove evidence of your merge conflicts

Comment (dane-johnson) [inline src/util/codemirror-utils.js]: How about just combining the two into one filter, instead of having 2 separate filters for True and False?

**Comment by dane-johnson** [comment]
How about just combining the two into one filter, instead of having 2 separate filters for True and False?

**Comment by Hopding** [comment]
@dane-johnson That cannot really be done until we make use of the Parser's tagging ability, because they appear as two different types of nodes in the tree.

**Comment by dane-johnson** [comment]
Can't you just make both nodes have the same key?

**Comment by Hopding** [comment]
Nope, because the key _is_ the token type.

**Comment by dane-johnson** [comment]
And the token type is literally just True? That's not really a token type it's the token itself.

**Comment by Hopding** [comment]
It is how it appears in the tree. It normally wouldn't appear at all, because it is a terminal node. Again though, this is something that can be changed when we update the UI to use the tagging ability of the parser.

**Comment by dane-johnson** [comment]
Maybe we should hold off on this merge until then. It seems silly to me to have a filter that literally just highlights one word that's already in the name of the filter.

**Comment by solkaz** [comment]
It's fine for merging as is. It does not affect performance or functionality in a negative way. If we want to refactor the code, fine, we can do that later, but stylistic changes shouldn't block a PR.

**Comment by dane-johnson** [comment]
☝️ I wholeheartedly disagree

**Pull request merged**